lib/parser/contents/classes/MusicResponsiveListItem.js

```diff
@@ -106,11 +111,17 @@ class MusicResponsiveListItem {
           
     const authors = this.#flex_columns[1].title.runs?.filter((run) => run.endpoint.browse?.id.startsWith('UC'));
 
-    authors && (this.authors = authors.map((author) => ({
-      name: author.text,
-      channel_id: author.endpoint.browse.id,
-      endpoint: author.endpoint
-    }))) && (this.author = this.authors[0]);
+	if (authors) {
+		this.authors = authors.map((author) => ({
+			name: author.text,
+			channel_id: author.endpoint.browse.id,
+			endpoint: author.endpoint
+		}))
+		// this seems to be always true?
```

Comment:
It is — 
You can completely remove the `author` property. It was added there for backward compatibility but since there's `authors` it may actually be a bit redundant.

Reply:
Okay!